wrappers/python/tests/TestUnitCell.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+import math
+from simtk.openmm import Vec3
+from simtk.unit import *
+from simtk.openmm.app.internal.unitcell import computePeriodicBoxVectors
+from simtk.openmm.app.internal.unitcell import computeLengthsAndAngles
+from simtk.openmm.app.internal.unitcell import reducePeriodicBoxVectors
+
+def strip_units(x):
+    if is_quantity(x): return x.value_in_unit_system(md_unit_system)
+    return x
+
+class TestAmberPrmtopFile(unittest.TestCase):
+
+    """Test the AmberPrmtopFile.createSystem() method."""
+
```

Comment:
The class name and doc string here are incorrect.  It should be TestUnitCell, not TestAmberPrmtopFile.